src/app/static/src/js/components/reportLog/runningReportsList.vue

```diff
@@ -58,10 +58,23 @@
             }
         },
         computed: {
+            initialInputValue(){
```

Comment:
This maybe shouldn't be a computed value if it's only every used to generate an initial query value on mount - could just move this code to `mounted` and set `query` from there directly.